rhv_checks.py

```diff
@@ -203,11 +203,42 @@ def check_datacenters_status(system, **kwargs):
         print("Ok: all datacenter(s) are in the OK state: {}".format(okay))
         sys.exit(0)
 
+
+def check_storage_domain_attached_status(system, **kwargs):
+    """ Check the usage of all the datastores on the host. """
+    okay, critical, all = [], [], []
+    storage_domains_service = system.api.system_service().storage_domains_service()
+    all_sd = storage_domains_service.list()
+    data_centers_service = system.api.system_service().data_centers_service()
+    all_dc = data_centers_service.list()
+
+    for dc in all_dc:
+        print("Checking for Data Center: {}".format(dc.name-dc.id))
```

Comment:
I'd say that this print statement is unnecessary since you are printing out information about the check down below. I don't think it will output anywhere that a user will be able to see it. This info would be good for a logger, but since we don't have one yet, I think it's best to remove the `print` statement.

Reply:
@john-dupuy Okay I will remove `print` statement. Which command output do you want to it here .?

Reply:
For the command output, just run the check `python check_rhv_main.py [args] -m storage_domain_attached`, do `python check_rhv_main.py -h` to see what args are necessary, I think it requires the manager address, and username and password.